CSS_files/components/image/image.css

```diff
@@ -0,0 +1,8 @@
+img{
+   width: 100%;
+   object-fit: contain; 
+}
+
+.img-circle{
+    border-radius: 999px;
```

Comment:
you can use border-radius: 50% or border-radius: 100%, to make it a circle instead of using px.

Reply:
Okay got it